### ISSUE TITLE
Please update cabal file on hackage to point to this issue tracker

### ISSUE THREAD

**Issue by thomie** [haskell/hoopl#14]
https://hackage.haskell.org/package/hoopl still mentions:

```
Bug tracker http://ghc.haskell.org/trac/ghc/newticket?component=libraries/hoopl
```

Please update it to point to https://github.com/haskell/hoopl/issues

**Comment by bgamari** [comment]
Done.

**Comment by bgamari** [comment]
Or rather, the Cabal file has been fixed (cca20ad0c2606db16dd548900aaaacc7ec601a55) but someone still needs to push to Hackage. Reopening.

**Comment by hvr** [comment]
Starting with https://hackage.haskell.org/package/hoopl-3.10.2.0 this seems to have been addressed
